cli/cmd/sudo/whitelist/add.go

```diff
@@ -19,11 +19,42 @@ func AddCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
+			// check the number of args to determine if the project is provided
+			if len(args) == 3 {
+				// project is not provided, this is an organization whitelist
+				org := args[0]
+				domain := args[1]
+				role := args[2]
+
+				ch.PrintfWarn("Warn: Whitelisting will give all users from domain %q access to the organization %q as %s\n", domain, org, role)
+				ok, err := cmdutil.ConfirmPrompt("Do you want to continue", "", false)
+				if err != nil {
+					return err
+				}
+				if !ok {
+					ch.PrintfWarn("Aborted\n")
+					return nil
+				}
+
+				_, err = client.CreateWhitelistedDomain(ctx, &adminv1.CreateWhitelistedDomainRequest{
+					Organization: org,
+					Domain:       domain,
+					Role:         role,
+				})
+				if err != nil {
+					return err
+				}
+
+				ch.PrintfSuccess("Success\n")
+			}
```

Comment:
Needs to return here – else it will try to run the project flow after the org flow